src/pretix/control/templates/pretixcontrol/order/index.html

```diff
@@ -18,7 +18,7 @@
         {% include "pretixcontrol/orders/fragment_order_status.html" with order=order class="pull-right" %}
     </h1>
     {% if 'can_change_orders' in request.eventpermset %}
-        {% if order.status == 'n' or order.status == 'p' or order.status == 'e' %}
+        {% if order.status == 'n' or order.status == 'p' or order.status == 'e' or order.status == 'c' %}
```

Comment:
Could you instead just drop this ``if`` clause alltogether? It doesn't to anything useful any more since all relevant buttons have their own if clause :)

Reply:
I think the reason I added the 'c' case to the `if` statement is that we still have the 'r' case out there.
If we're ok with the 'r' case having the same two button behavior as 'c' (and why not), then I'll make the change.

Reply:
Yep, I think ``r`` should have the same behavior as ``c``